pom.xml

```diff
@@ -1100,7 +1100,7 @@
             <dependency>
                 <groupId>com.microsoft.sqlserver</groupId>
                 <artifactId>mssql-jdbc</artifactId>
-                <version>9.4.1.jre11</version>
+                <version>10.2.0.jre11</version>
```

Comment:
Update SQL server JDBC driver to 10.2.0 **_for Java 11_**

Do you mean it was not compatible with Java 11 before this commit?

Reply:
It was .. but this is the new release for Java 11 .. if we want to update to us Java 17 .. there is also a 10.2.0.jre17 artifact .. but I believe that needs Java 17 and at this stage we still support Java 11. But once we switch to Java 17 we can use this version of the driver.

Reply:
The current commit message is misleading. I would recommend removing the phrase.

Reply:
Thanks @ebyhr , i was confused too. I edited the PR title, but commit message still needs to be fixed.